src/core/continuous_space.jl

```diff
@@ -215,13 +215,24 @@ end
 # filter!(...) where the filtering function checks distances w.r.t. `r`.
 """
     space_neighbors(pos::Tuple, model::ABM, r::Real)
-Return neighbours of a particular position `pos`, within radius `r` for any `SpaceType`.
+Return IDs of all agents within radius `r` from a particular position `pos` for any `SpaceType`.

```

Comment:
`SpaceType` can be missleading: this is not a type defined in Agents.jl. You should replace it with just the english word "space", without back ticks. 

In general if something is in backticks, it is implied that it exists in code, either as given argument, or as exported name by the module.

Reply:
Ok. `SpaceType` was there before I edit this function.